newIDE/app/src/InstancesEditor/index.js

```diff
@@ -804,16 +811,72 @@ export default class InstancesEditor extends Component<Props> {
     this.viewPosition.scrollTo(x, y);
   }
 
-  centerView() {
+  zoomToFitContent() {
+    const instanceMeasurer = this.instancesRenderer.getInstanceMeasurer();
+    let contentAABB: ?Rectangle;
+    const getInstanceRectangle = new gd.InitialInstanceJSFunctor();
+    // $FlowFixMe - invoke is not writable
+    getInstanceRectangle.invoke = instancePtr => {
+      // $FlowFixMe - wrapPointer is not exposed
+      const instance = gd.wrapPointer(instancePtr, gd.InitialInstance);
+      if (!contentAABB) {
+        contentAABB = instanceMeasurer.getInstanceAABB(
```

Comment:
Not really, because if you create it outside, it will have 0 as default values for the 4 coordinates. So if you have a single instance that fits in [10, 10, 10, 10], it will end up with [0, 0, 10, 10]